config/pepperConfig.js.ctmpl

```diff
@@ -105,4 +105,20 @@ var DDP_ENV = {
   {{end}}
 {{end}}
 
+{{if eq $study_guid "testboston"}}
```

Comment:
@fat23cat you  read my mind--ignore my  comment in the backend PR.  I see you've already handled it :)

Reply:
@fat23cat I have the study _guid_ as uppercase `TESTBOSTON` in the study-server branch https://github.com/broadinstitute/ddp-study-server/pull/192. Should I change it to lowercase?

Reply:
@zyme, should we use uppercase or lowercase for the study guid? For now, other study guids names register is not consistent.

Reply:
Lowercase it is! I'll change in other PR.